.github/workflows/mshv-tests.yaml

```
@@ -0,0 +1,97 @@
+name: Build & Test MSHV Crate
+on: 
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to build and test'
+        required: true
+        default: 'main'
+jobs:
+  infra-setup:
+    name: MSHV Infra Setup
```

```suggestion
    name: MSHV Infra Setup (x86_64)
```

Perhaps we should name it like this?

---

Sure

---

Yeah, it will be separate. We can name it with this convention.